.env.example

```
@@ -0,0 +1,8 @@
+#env.example
+OPENDOTA_BASE_URL=https://api.opendota.com/api
+OPENDOTA_RATE_LIMIT=60
+OPENDOTA_TIMEOUT=30
+OPENDOTA_RETRIES=3
+
+DB_PATH=services/ingestion/db/data.sqlite
+LOG_LEVEL=INFO
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)